docs/tutorials/creating-a-runtime-module.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Creating a Runtime Module"
+title: "Creating a Runtime Pallet"
```

Comment:
AFAIK it should be `Runtime Module` -> `Pallet` rather than `Runtime Pallet`.

Can you confirm with @gavofyork in some public place?

Reply:
What did he say?